raiden/tests/integration/api/test_pythonapi_regression.py

```diff
@@ -33,26 +33,29 @@ def test_close_regression(raiden_network, deposit, token_addresses):
     amount = PaymentAmount(10)
     identifier = PaymentID(42)
     secret, secrethash = factories.make_secret_with_hash()
-    assert api1.transfer_and_wait(
-        registry_address=registry_address,
-        token_address=token_address,
-        amount=amount,
-        target=TargetAddress(api2.address),
-        identifier=identifier,
-        secret=secret,
-        transfer_timeout=10,
-    )
-    exception = ValueError("Waiting for transfer received success in the WAL timed out")
-    with gevent.Timeout(seconds=5, exception=exception):
+    timeout = block_offset_timeout(app1.raiden)
+    with timeout:
+        assert api1.transfer_and_wait(
+            registry_address=registry_address,
+            token_address=token_address,
+            amount=amount,
+            target=TargetAddress(api2.address),
+            identifier=identifier,
+            secret=secret,
+        )
+        timeout.exception_to_throw = ValueError(
```

Comment:
can you please give this exception as the parameter to `block_offset_timeout`

Reply:
It is set later so the error message would tell us which part of the `with` block timed out. The same happens in three other places in this PR, I hope the state altering of the timeout is ok. (If not I'll just change all of them with copied block timeouts, same block number, different exceptions.) I added an error message parameter to the `block_offset_timeout` call now to clarify.